api/courses.js

```diff
@@ -51,8 +64,8 @@ router.get('/', async (req, res, next) => {
  * requireAdmin - must have Admin Role
  *
  */
-router.post('/', requireAuthentication, requireAdmin, async (req, res, next) => {
- if (validateAgainstSchema(req.body, CourseSchema)) {
+router.post('/', /*requireAuthentication, requireAdmin,*/ async (req, res, next) => {
```

Comment:
Might need to encorporate that auth in production

Reply:
When I did the merge it looks like it pulled an older version of this file. The new commit should have the current version of the file